cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -67,8 +68,21 @@
     private boolean helpFlag;
 
     private static boolean isDirEmpty(final Path directory) throws IOException {
+
         try (DirectoryStream<Path> dirStream = Files.newDirectoryStream(directory)) {
-            return !dirStream.iterator().hasNext();
+            //Check whether the OS is MacOS and the folder contains .DS_Store file
+            Iterator pathIterator = dirStream.iterator();
+            if (!pathIterator.hasNext()) {
+                return true;
+            }
+            Path path = (Path) pathIterator.next();
+            Path filePath = path.getFileName();
```

Comment:
Shall we change this variable name to 'fileName' since we are referring to the file name?